sklearn/gaussian_process/kernels.py

```diff
@@ -1861,3 +1861,181 @@ def is_stationary(self):
     def __repr__(self):
         return "{0}(gamma={1}, metric={2})".format(
             self.__class__.__name__, self.gamma, self.metric)
+
+
+class SelectDimensionKernel(Kernel):
+    """Wrapper for applying kernels on selected active dimensions of input data.
+
+    Parameters
+    ----------
+    kernel : a kernel object.
+        an instantiation of kernel object.
+
+    active_dims: array-like, shape (n_active_dims,) or (n_features,)
```

Comment:
Sorry, I do not understand. What does "source object's to extract without copying" mean?